### ISSUE TITLE
Version 2.2 doesn't work on NVDA 2020.4

### ISSUE THREAD

**Issue by dreinn** [mltony/nvda-browser-nav#6]
Hello,
I'm using version 2020.4 of NVDA because on my office I haven't the admin rights to update NVDA.

I downloaded Browsernav, it is enabled from the addon list but it appears that no keystrokes are active.
I found this error on the log file:
ERROR - globalPluginHandler.listPlugins (16:24:27.750) - MainThread (6708):
Error importing global plugin browserNav
Traceback (most recent call last):
  File "globalPluginHandler.pyc", line 23, in listPlugins
  File "importlib\__init__.pyc", line 127, in import_module
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "D:\Users\MI09424\AppData\Roaming\nvda\addons\browsernav\globalPlugins\browserNav\__init__.py", line 58, in <module>
    from . import quickJump
  File "D:\Users\MI09424\AppData\Roaming\nvda\addons\browsernav\globalPlugins\browserNav\quickJump.py", line 463, in <module>
    globalConfig  = loadConfig()
  File "D:\Users\MI09424\AppData\Roaming\nvda\addons\browsernav\globalPlugins\browserNav\quickJump.py", line 450, in loadConfig
    return QJConfig(json.loads(rulesConfig))
  File "D:\Users\MI09424\AppData\Roaming\nvda\addons\browsernav\globalPlugins\browserNav\quickJump.py", line 414, in __init__
    for item in d['sites']
  File "D:\Users\MI09424\AppData\Roaming\nvda\addons\browsernav\globalPlugins\browserNav\quickJump.py", line 414, in <listcomp>
    for item in d['sites']
  File "D:\Users\MI09424\AppData\Roaming\nvda\addons\browsernav\globalPlugins\browserNav\quickJump.py", line 361, in __init__
    for bookmarkDict in d['bookmarks']
  File "D:\Users\MI09424\AppData\Roaming\nvda\addons\browsernav\globalPlugins\browserNav\quickJump.py", line 361, in <listcomp>
    for bookmarkDict in d['bookmarks']
  File "D:\Users\MI09424\AppData\Roaming\nvda\addons\browsernav\globalPlugins\browserNav\quickJump.py", line 307, in __init__
    for attrDict in d['attributes']
  File "D:\Users\MI09424\AppData\Roaming\nvda\addons\browsernav\globalPlugins\browserNav\quickJump.py", line 307, in <listcomp>
    for attrDict in d['attributes']
  File "D:\Users\MI09424\AppData\Roaming\nvda\addons\browsernav\globalPlugins\browserNav\quickJump.py", line 255, in __init__
    object.__setattr__(self, 'attribute', QJAttribute(d['attribute']))
  File "D:\Users\MI09424\AppData\Roaming\nvda\addons\browsernav\globalPlugins\browserNav\quickJump.py", line 183, in __init__
    value = controlTypes.Role(value)
AttributeError: module 'controlTypes' has no attribute 'Role'

Can you fix it or version 2020.4 is not supported?
